rules/relational-database-design/rule.md

```diff
@@ -26,13 +26,14 @@ These problems can be mitigated using Entity-Relationship Diagrams (ERDs) and Da
 Database Normalization is a systematic approach to designing databases that reduces redundancy and avoids potential anomalies during data update, deletion, or insertion. This process involves organizing the columns (attributes) and tables (relations) of a database to ensure their dependencies are properly enforced by database integrity constraints.
 
 Normalization is typically carried out through a series of steps called normal forms, each with specific rules:
-•	First Normal Form (1NF)
-•	Second Normal Form (2NF)
-•	Third Normal Form (3NF)
+- First Normal Form (1NF)
+- Second Normal Form (2NF)
+- Third Normal Form (3NF)
 
 There are [further normal forms](https://en.wikipedia.org/wiki/Database_normalization), and the theory constantly evolves, but these 3 are the bare minimum required for database design.
 
 `youtube: https://youtu.be/J-drts33N8g`
```

Comment:
Recommend line break after the YouTube video to ensure that the Figure caption is on the next line.